src/lib.rs

```diff
@@ -231,12 +233,24 @@ cfg_if! {
                 #[path = "wasm32_bindgen.rs"] mod imp;
             } else if #[cfg(feature = "stdweb")] {
                 #[path = "wasm32_stdweb.rs"] mod imp;
-            } else {
+            } else if #[cfg(any(feature = "wasm_dummy", feature = "dummy"))] {
                 #[path = "dummy.rs"] mod imp;
+            } else {
+                compile_error!("\
+                    wasm32-unknown-unknown target requires one of the following
+                    features to be enabled: `stdweb`, `wasm-bindgen` or
+                    `dummy`/`wasm_dummy`\
+                ");
             }
         }
-    } else {
+    } else if #[cfg(feature = "dummy")] {
         #[path = "dummy.rs"] mod imp;
+    } else {
+        compile_error!("\
+            target is not supported, you may enable dummy implementation \
+            using `dummy` feature or replace `getrandom` crate with a custom \
```

Comment:
I'd remove/rephrase the reference to the custom crate. Right now the "replace" message is a little confusing.